package.json

```diff
@@ -7,8 +7,9 @@
     "dev-migration": "npx knex migrate:latest --env=development",
     "server": "npm run dev-migration && nodemon index.js",
     "migration": "npx knex migrate:latest --env=production",
+    "migration-test": "npx knex migrate:latest --env=test",
     "start": "npm run migration && node index.js",
-    "test": "cross-env DB_ENV=test nyc --reporter=html --reporter=text --reporter=lcov jest --verbose --forceExit",
+    "test": "npm run migration-test && nyc --reporter=html --reporter=text --reporter=lcov jest --verbose --forceExit",
```

Comment:
Hey man, I'm thinking since we have this command `npm run migration-test` on Travis config, is there a reason why we changed the test script?

Reply:
@ogwurujohnson Created the script to migrate the table before the test runs when developing in case of a change in migration. Because during development, the script in the `travis.yml` can't be accessed